src/components/UI/Sidebar/components/SidebarUtils/index.tsx

```diff
@@ -1,30 +1,41 @@
-import React from 'react'
-
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faPatreon, faDiscord } from '@fortawesome/free-brands-svg-icons'
+import { faDiscord, faPatreon } from '@fortawesome/free-brands-svg-icons'
 import {
   faCoffee,
-  faUser,
   faDoorOpen,
+  faUser,
   faUserAlt,
   faWineGlass
 } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import ElectronStore from 'electron-store'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { NavLink, useHistory } from 'react-router-dom'
+import { handleQuit, openDiscordLink } from 'src/helpers'
+import ContextProvider from 'src/state/ContextProvider'
+import './index.css'
 
 const { ipcRenderer } = window.require('electron')
 const Store = window.require('electron-store')
+
 const configStore: ElectronStore = new Store({
   cwd: 'store'
 })
 const gogStore = new Store({
   cwd: 'gog_store'
 })
-import { handleQuit } from 'src/helpers'
-import './index.css'
-import { openDiscordLink } from 'src/helpers'
-import { useTranslation } from 'react-i18next'
-import ContextProvider from 'src/state/ContextProvider'
-import { NavLink, useHistory } from 'react-router-dom'
+
+const QuitButton: React.FC = () => {
```

Comment:
Maybe we could add this component to its own file to avoid having more than one component declared in the same file 👍🏽

Reply:
Moved